platform/security/docs/src/main/resources/ManContents.adoc

```diff
@@ -1,120 +1,109 @@
-* adding the application's feature repository back into DDF and installing its main feature, which typically is of the form `<applicationName>-app`, e.g., `catalog-app`.
+|Security STS Client
+|`ddf.security.sts.client.configuration`
+|Settings for STS client
 
-.Reverting DDF application's uninstall
-[source,terminal,linenums]
-----
-features:addurl <DDF application's feature repository URL>
-features:install <DDF application's main feature>
-
-Example:
+|Security STS Guest Claims Handler
+|`ddf.security.sts.guestclaims`
+|Add or remove attributes to be attached to claims for guest users.
 
-    ddf@local>features:addurl mvn:ddf.catalog/catalog-app/2.3.0/xml/features
-    ddf@local>features:install catalog-app
-----
+|Security STS Guest Validator
+|`ddf.security.sts.guestvalidator`
+|Configure realms to use with Guest Validator
 
-=== Upgrade
+|Security STS PKI Token Validator
+|`org.codice.ddf.security.validator.pki`
+|Configure realms to use with PKI Token Validator
 
-To upgrade an application, complete the following procedure.
+|Security STS Property File Claims Handler
+|`org.codice.ddf.security.sts.claims.property.PropertyFileClaimsHandler`
+|Settings for retrieving claims from properties file
 
-. Uninstall the application by following the Uninstall Applications instructions above.
-. Install the new application KAR file by copying the admin-app-X.Y.kar file to the `<INSTALL_DIRECTORY>/deploy` directory.
-. Start the application. +
-`features:install admin-app`
-. Complete the steps in the Verify section above to determine if the upgrade was successful.
+|Security STS Server
+|`ddf.security.sts`
+|Settings for STS Server
 
-== Configuration
+|Security STS WSS
+|`ddf.security.sts.wss.configuration`
+|WSS-enabled version of STS
 
-This component can be configured using the normal processes described in the 
-Configuration section. Within the pages for each of the applications are specific instructions on the configurations for the bundles and any additional information that may help decide how the configuration should be set for use cases.
+|Security Simple AuthZ Realm
+|`ddf.security.pdp.realm.SimpleAuthzRealm`
+|Configuration of Match-One and/or Match-All mappings in SimpleAuthz realme
 
-== Whitelist
+|Session
+|`org.codice.ddf.security.filter.login.Session`
+|Set session timeout
 
-The following packages have been exported by the DDF Security application and are approved for use by third parties:
+|Web Context Policy Manager
+|`org.codice.ddf.security.policy.context.impl.PolicyManager`
+|Configure Realms, Auth types, and Attributes for different contexts.
 
-* ddf.security.expansion
-* ddf.security.sts.client.configuration
-* ddf.security.common.callback
-* ddf.security.common.util
+|===
 
-== Applications
+==== Applications Included in {branding} Security
 
-* Security Core
 * Security CAS
+* Security Core
 * Security Encryption
 * Security PEP
 * Security PDP
 * Security STS
+* Security IdP
```

Comment:
Was this meant to be an alphabetically ordered list? if so IdP should have been after `Security Encryption`